src/Beacon.jsx

```diff
@@ -137,10 +159,13 @@ export class Beacon extends Component {
     }
   }
 
-  getVerticalAttachment(bounds) {
+  getVerticalAttachment(bounds, horizontalAttachment) {
```

Comment:
Need to take into account the horizontal attachment since we don't want it to be centered on both axes (the tooltip should never be on top of the tether element).